src/ontology/hp-edit.owl

```diff
@@ -87607,16 +88110,22 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0008757> "Unil
 EquivalentClasses(<http://purl.obolibrary.org/obo/HP_0008757> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000763> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/UBERON_0003706>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000634>))))
 SubClassOf(<http://purl.obolibrary.org/obo/HP_0008757> <http://purl.obolibrary.org/obo/HP_0001605>)
 
-# Class: <http://purl.obolibrary.org/obo/HP_0008760> (Violent behavior)
+# Class: <http://purl.obolibrary.org/obo/HP_0008760> (Violent behaviour)
```

Comment:
UK spelling?

Reply:
This should be "behavior". It looks like the synonyms include both spellings.